src/main/java/com/example/springbootapp/service/EmployeeService.java

```diff
@@ -2,23 +2,27 @@
 
 import com.example.springbootapp.dto.EmployeeDTO;
 import com.example.springbootapp.entity.Employee;
+import com.example.springbootapp.exception.EmployeeExceptionHandler;
 import com.example.springbootapp.exception.EmployeeNotFoundException;
 import com.example.springbootapp.exception.EmployeeNotSavedException;
+import com.example.springbootapp.exception.EmployeeNotUniqueException;
 import com.example.springbootapp.mapper.EmployeeMapper;
 import com.example.springbootapp.repository.EmployeeRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
 import java.util.List;
-import java.util.Optional;
 import java.util.stream.Collectors;
 
 @Service
 @Transactional(readOnly = true)
 public class EmployeeService {
     private final EmployeeRepository employeeRepository;
     private final EmployeeMapper employeeMapper;
+    private static final Logger LOGGER = LoggerFactory.getLogger(EmployeeExceptionHandler.class);
```

Comment:
Please, constants should be first in class and add empty line to separate constants and fields

Reply:
Formatting is checked in whole project.